pkgs/games/starsector/default.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, alsa-lib
+, fetchzip
+, libXxf86vm
+, makeWrapper
+, openjdk8
+, stdenv
+, xorg
+}:
+
+let
+  openjdk = openjdk8;
+in
+stdenv.mkDerivation rec {
+  pname = "starsector";
+  version = "0.95a-RC15";
+
+  src = fetchzip {
+    url = "https://s3.amazonaws.com/fractalsoftworks/starsector/starsector_linux-${version}.zip";
+    sha256 = "sha256-/5ij/079aOad7otXSFFcmVmiYQnMX/0RXGOr1j0rkGY=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = with xorg; [
+    alsa-lib
+    libXxf86vm
+  ];
+
+  doBuild = false;
+
+  # need to cd into $out in order for classpath to pick up correct jar files
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r ./* $out
+    rm -r $out/jre_linux # remove jre7
+    rm $out/starfarer.api.zip
+
+    wrapProgram $out/starsector.sh \
+      --prefix PATH : ${openjdk}/bin \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath buildInputs} \
+      --run "mkdir -p \$XDG_DATA_HOME/starsector; cd $out"
+    ln -s $out/starsector.sh $out/bin/starsector
+  '';
+
+  # it tries to run everything with relative paths, which makes it CWD dependent
+  # also point mod, screenshot, and save directory to $XDG_DATA_HOME
+  prePatch = ''
```

```suggestion
  postPatch = ''
```
If you still know from where you got this please also fix this up.

---

I got this from https://discourse.nixos.org/t/game-got-starsector-to-run-on-nixos/5419